privval/socket_dialers_test.go

```diff
@@ -9,10 +9,19 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmnet "github.com/tendermint/tendermint/libs/net"
 )
 
+// GetFreeLocalhostAddrPort returns a free localhost:port address
+func GetFreeLocalhostAddrPort(t *testing.T) string {
```

Comment:
Could we unexport this?

Maybe also: `t.Helper()`

Reply:
I mean it's in a test file, so it's effectively unexported, but sure.

Reply:
Yeah, I ask not because it makes a functional difference, but because when my eye sees an exported thing I have to think about where else it might be used. 🙂